internal/api/v1/namespace/create.go

```diff
@@ -38,6 +38,7 @@ func (oc Controller) Create(c *gin.Context) apierror.APIErrors {
 		return apierror.InternalError(err)
 	}
 	if exists {
+		// TODO: Don't retry when this happens (on the cli)
```

Comment:
This was a reminder, irrelevant to this PR. We still need to fix this one though. Split to another issue?

Reply:
Follow up issue: https://github.com/epinio/epinio/issues/1081